games/static/games/proto.js

```diff
@@ -67,31 +107,52 @@ class Board{
         this.width = width
         this.aliens = []
         this.squares = []
-        const alienInvaders = [
-        0, 1, 2, 3, 4, 5, 6, 7, 8, 9,
-        15, 16, 17, 18, 19, 20, 21, 22, 23, 24,
-        30, 31, 32, 33, 34, 35, 36, 37, 38, 39 
-        ]    
+        
+    }
+    createChildSquareAddToSquares(index) {
+        let div = document.createElement('div') 
+        this.parent.appendChild(div)
+        this.squares.push(new Square(index, div, this.width)) 
+        // return ?
+    }
+    draw(setup) {
+        let alienLocations = setup.getAlienLocations()
         for(let i = 0; i < width*width; i++) {
-            let div = document.createElement('div') 
-            this.parent.appendChild(div)
-            this.squares.push(new Square(i, div)) 
-
-            if (alienInvaders.includes(i)) {
-                let invader = createInvader(alienInvaders[i], width)
-                invader.draw(square)
-                this.aliens.push(invader)
-            } else if (i > width*width-width) {
-                square.classList.add('ground')
-            }
+            this.createChildSquareAddToSquares(i)
```

Comment:
maybe something like `createBoardChildSquare` or just `createChildSquare`?
isnt child indicates that its already added ?